commerce/domain/orchestrators/classes/CartCalculateSample.cls

```diff
@@ -3,65 +3,84 @@
 // for orchestrators and then the extension must be linked to the webstore via the appropriate setup.
 
 // This class must extend the CartExtension.CartCalculate class
-public class CartCalculateSample extends CartExtension.CartCalculate {
+global class CartCalculateSample extends CartExtension.CartCalculate {
+
+    /**
+     * @description Default constructor
```

Comment:
Please add to comment that "All classes extending CartExtension.CartCalculate must have default constructor defined'